hack/test-go.sh

```diff
@@ -195,8 +195,7 @@ runTests() {
 }
 
 runTestsForVersion() {
-  export KUBE_API_VERSION="$1"
-  runTests "${@:2}"
+  KUBE_API_VERSION="$1" runTests "${@:2}"
```

Comment:
Nice!

I think using `shift` ends up getting cleaner, especially if you end up changing the code later...

I'd do something like

```shell
runTestsForVersion() {
  local api_version=$1
  shift
  KUBE_API_VERSION="${api_version}" runTests "$@"
}
```

But what you have is certainly fine.